app/models/institutional-user.ts

```diff
@@ -0,0 +1,21 @@
+import { attr, belongsTo } from '@ember-decorators/data';
+import DS from 'ember-data';
+
+import UserModel from 'ember-osf-web/models/user';
+import OsfModel from './osf-model';
+
+export default class InstitutionalUserModel extends OsfModel {
+    @attr('fixstring') department?: string;
+    @attr('number') publicProjectCount!: number;
+    @attr('number') privateProjectCount!: number;
```

Comment:
We are also going to cache the user's full name here so we don't have to pull the user relationship to get it.:
```ts
@attr('fixstring') userFullName: string;
```